lib/store_web/templates/starbook/index.html.eex

```diff
@@ -0,0 +1,60 @@
+<div class="starbook">
+  <h1>StarBook <small>Mk V</small></h1>
+  <p>with elementary OS</p>
+  <div>
+    <a href="https://starlabs.systems/products/starbook?variant=39599067136164&rfsn=4227837.e8f025" class="button suggested">Buy Now</a>
+  </div>
+
+  <section id="stealth">
+    <img alt="StarBook Mk V" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/stealth.png") %>" />
+    <h3>Stealth power.</h3>
+    <p>StarBook Mk V looks unassuming in matte black anodized aluminum, but it’s engineered to pack a punch. 11th-gen Intel Core processors deliver performance on demand while sipping power. Low-latency memory keeps everything snappy. And the over-provisioned solid state StarDrive offers incredible speeds for opening your apps and files.</p>
+  </section>
+
+  <section id="display">
+    <img alt="StarBook Mk V" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/hinge.png") %>" />
+    <div>
+      <h3>Matte or glossy—why not both?</h3>
+      <p>The 14-inch ARC display features a protective hard coating that lets color shine through like glass while diffusing ambient light like a matte display. Combined with IPS technology and 400 nits of brightness, the display is visible in nearly any light and angle. Which is convenient, because the 180° hinge lets you get some pretty wild angles.</p>
+    </div>
+  </section>
+
+  <section id="trackpad">
+    <div>
+      <h3>Slip and swipe.</h3>
+      <p>Glide across the 5.5-inch-diagonal glass trackpad for precise and smooth multi-touch gestures. Scroll through documents or web pages, fly through your workspaces, and swipe to go back throughout elementary OS—all with 1:1 gestures that perfectly track your fingers.</p>
+    </div>
+    <img alt="StarBook Mk V" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/trackpad.png") %>" />
```

Comment:
Seems like there's a bit of stuff on the corners here you can see in dark mode

Reply:
Yeah, good catch, will replace.